sonar-project.properties

```diff
@@ -0,0 +1,8 @@
+sonar.projectKey=broadinstitute_pearl
```

Comment:
Will this need to change if we rename the repo?

Reply:
I talked to Tom about this -- some stuff will need to change, but it shouldn't be a big deal

Reply:
^^ that's right